libs/rest/src/paths.ts

```diff
@@ -0,0 +1,67 @@
+import trimEnd from 'lodash/trimEnd';
+import path from 'path';
+import { getBaseUrl } from './rest-config';
+import tradingDataApi from '@vegaprotocol/rest-clients/openapi/trading_data_v2.openapi.json';
```

Comment:
Good idea to use this

Reply:
Yeah, I can generate additional file there that would consist of only the paths.